setuptools/tests/test_config.py

```diff
@@ -391,6 +391,23 @@ def test_classifiers(self, tmpdir):
         with get_dist(tmpdir) as dist:
             assert set(dist.metadata.classifiers) == expected
 
+    def test_deprecated_config_handlers(self, tmpdir):
+        fake_env(
+            tmpdir,
+            '[metadata]\n'
+            'version = 10.1.1\n'
+            'description = Some description\n'
+            'requires = some, requirement\n'
+        )
+
+        with pytest.raises(DeprecationWarning):
```

Comment:
```suggestion
        with pytest.warns(DeprecationWarning):
```

Reply:
Oops. saw this now. I did not realize this. I am using pytest.deprecated_call() instead which I believe is equivalent.